local-interchain/interchain/handlers/container_log_stream.go

```diff
@@ -0,0 +1,153 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"unicode"
+
+	dockertypes "github.com/docker/docker/api/types"
+	dockerclient "github.com/docker/docker/client"
+	"github.com/strangelove-ventures/interchaintest/v8/chain/cosmos"
+	"go.uber.org/zap"
+)
+
+var removeColorRegex = regexp.MustCompile("\x1b\\[[0-9;]*m")
```

Comment:
Is this necessary because certain operating systems or terminals cannot render the colors in the logs?

Reply:
Since this is being forwarded over a REST api, it is always the case. So most users parsing that data don't care about. We may in the future want to parse out the colors which is why it is possible to opt out and see those ASNI escape sequences